### ISSUE TITLE
Bookworm doesn't recognize headings in PDF documents

### ISSUE THREAD

**Issue by Cleversn** [blindpandas/bookworm#28]
## Describe the Problem

In some PDF documents, it is possible to navigate through headings using the NVDA commands h and shift-h, just like in HTML web pages. However, this currently doesn't work in Bookworm; it only works in Adobe Acrobat Reader DC.

## To Reproduce

1. Open the Reaper Users' Guide in Acrobat Reader DC; see link below to download the file.
2. Go to a page that contains headings, for example 287.
3. Using NVDA, press h and shift-h. You'll notice that there are two headings, namely 15.1 and 15.2.
4. Open the same PDF file in Bookworm, go to the same page, and notice that pressing h and shift-h does nothing.

## Expected behavior

Bookworm should recognize the same headings as Acrobat Reader does.

## The File You've Opened

Please download it, as it's too big to attach here:
https://www.reaper.fm/userguide/ReaperUserGuide5985c.pdf

## Desktop (please complete the following information):

 - Windows 10 pro 64-bit 1909.
 - Bookworm 0.1B3

**Comment by mush42** [comment]
Hi @Cleversn,

Currently we do not support book navigation using structural elements, e.g. headings.

I will keep this issue open just in case we started to add this feature.

**Comment by mush42** [comment]
Unfortunately, implementing this feature is not possible using the technologies we currently use.

We'll re-open this issue when a fix is possible.

Regards,
